tool/jt.rb

```diff
@@ -743,7 +743,7 @@ def rebuild(*options)
         vm_args << '--exceptions-print-uncaught-java=true'
       when '--infopoints'
         vm_args << '--vm.XX:+UnlockDiagnosticVMOptions' << '--vm.XX:+DebugNonSafepoints'
-        vm_args << '--engine.EnableInfopoints'
+        vm_args << '--vm.Dgraal.TruffleEnableInfopoints=true'
```

Comment:
I think we don't use `--infopoints` anymore, probably we can just remove it now.